File: 2.0/js/calls/sendCallsSounds.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('api-login', 'api-password');

client.ready(() => {
    client.calls.sendCalls({
      body: [
        {
          phoneNumber: '12135551100',
          liveMessage: 'Hello, Alice, this is message for live answer',
          machineMessage: 'Hello, Alice, this is message for answering machine'
        },
        {
          phoneNumber: '12135551101',
          liveMessage: 'Hello, Bob, this is message for live answer',
          machineMessage: 'Hello, Bob, this is message for answering machine'
        }
      ]
    })
      .then((response) => {
        console.log(response.obj);
      })
      .catch((err) => {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/calls/sendCallsSounds.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.calls.sendCalls(
    body=[
        {
            'phoneNumber': '12135551100',
            'liveMessage': 'Hello, Alice, this is message for live answer',
            'machineMessage': 'Hello, Alice, this is message for answering machine'
        },
        {
            'phoneNumber': '12135551101',
            'liveMessage': 'Hello, Bob, this is message for live answer',
            'machineMessage': 'Hello, Bob, this is message for answering machine'
        }
    ]
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)